pkgs/development/python-modules/pytest-astropy/default.nix

```diff
@@ -39,6 +39,7 @@ buildPythonPackage rec {
 
   # pytest-astropy is a meta package and has no tests
   doCheck = false;
+  dontUsePytestCheck = true;
```

Comment:
This shouldn't be required and shows us that some dependency is propagating pytestCheckHook.

Reply:
I found one propagating it.